### PR TITLE
[NETBEANS-2472] Mark overriding/overridden constants and properties as annotations

### PR DESCRIPTION
**Pull request by junichi11**
https://issues.apache.org/jira/browse/NETBEANS-2472

netbeans-2472-overriding-fields-and-constants

### CODE REVIEW

File: ide/csl.api/src/org/netbeans/modules/csl/editor/overridden/Bundle.properties [hunk 1]
```diff
@@ -19,7 +19,7 @@ CAP_Implements=Implements
 CAP_Overrides=Overrides
 #{0}: implements/overrides what
 TP_Implements=Implements method from : {0}
-TP_Overrides=Overrides method from: {0}
+TP_Overrides=Overrides from: {0}
```
Comment (junichi11) [inline ide/csl.api/src/org/netbeans/modules/csl/editor/overridden/Bundle.properties:22]: @sdedic Is this change OK? Constants and fields are overridden in PHP.

Comment (sdedic) [reply ide/csl.api/src/org/netbeans/modules/csl/editor/overridden/Bundle.properties:22]: I guess so ?

Comment (junichi11) [reply ide/csl.api/src/org/netbeans/modules/csl/editor/overridden/Bundle.properties:22]: Thanks!

**Comment by junichi11** [comment]
@tmysik Could you please have a look at this?

**Review by tmysik — APPROVED** [review]

**Comment by tmysik** [comment]
@junichi11 Looks good to me, approved. Thanks!

**Comment by junichi11** [comment]
@tmysik Thank you, Tomas! Will merge.

**Comment by tmysik** [comment]
@junichi11 You are welcome, of course.

**Pull request merged**